alwaysontop/main/index.js

```diff
@@ -114,16 +119,26 @@ const showAot = () => {
  */
  const hideAot = () => {
     logInfo('hide aot handler');
-    if (isIntersecting) {
-        logInfo('is intersecting. hiding');
+
+    if (undockedWindow || isIntersecting) {
         hideWindow();
     }
 };
 
+/**
+ * Attaches event handlers on the undocked window
+ */
+const addUndockedWindowHandlers = () => {
+    logInfo(`adding undocked window event handlers`);
+
+    undockedWindow.on('minimize', showAot);
+    undockedWindow.on('restore', hideAot);
```

Comment:
don't we need to consider blur/focus on the undocked window? eg, the undocked window always minimizes if you switch to other apps?

Reply:
we dont want to potentially have both the undocked and the aot window side by side. This is similar to what other apps are doing when the meeting is a separate window than the main app.